rustls/src/error.rs

```diff
@@ -634,7 +634,11 @@ mod other_error {
     ///
     /// Enums holding this type will never compare equal to each other.
     #[derive(Debug, Clone)]
-    pub struct OtherError(#[cfg(feature = "std")] pub Arc<dyn StdError + Send + Sync>);
+    pub struct OtherError(
+        #[cfg(not(feature = "withrcalias"))]
+        #[cfg(feature = "std")]
```

Comment:
Should merge double `cfg` attributes everywhere.

Why disable this for `withrcalias`?

Reply:
I recall using Rc led to an issue with this due to lack of `Send + Sync` marker traits.